packages/salesforcedx-vscode-apex/test/vscode-integration/commands/forceApexExecute.test.ts

```diff
@@ -180,6 +181,12 @@ describe('Force Apex Execute', () => {
         AnonApexLibraryExecuteExecutor.diagnostics,
         'set'
       );
+      debugLogsfolder = sb.stub(
+        projectPaths,
+        'debugLogsFolder'
+      ).get(() => {
+        return '.sfdx/tools/debug/logs';
+      });
```

Comment:
I have to look it up, but is .get specific to a getter stub?  Is so we'll want to use the `.returns` here since it just a method call verse a getter for a property (I think)

Reply:
Yess, thank you for calling that out! I had never used sinon (?), but I did some research and yup it replaces a new getter for the stub  :)